src/status_im/theme/core.cljs

```diff
@@ -1,8 +1,14 @@
 (ns status-im.theme.core
```

Comment:
this is legacy namespace why would we add fx here?

Reply:
I can move this to status-im2.common.theme.core, but then we also need to import `[quo.theme :as quo.theme]`,
there. Will that be ok?